tools/terraform/ecs.tf

```diff
@@ -76,6 +76,12 @@ resource "aws_ecs_cluster" "fleet" {
   }
 }
 
+resource "aws_ecs_cluster" "osquery-perf" {
+#  count = var.loadtesting ? 1 : 0
```

Comment:
count didn't seem to work as I expected everywhere, so I commented it out. 

eg:

```
╷
│ Error: Missing resource instance key
│
│   on ecs.tf line 282, in resource "aws_ecs_task_definition" "osquery-perf":
│  282:           awslogs-group         = aws_cloudwatch_log_group.osquery-perf.name
│
│ Because aws_cloudwatch_log_group.osquery-perf has "count" set, its attributes must be accessed on specific instances.
│
│ For example, to correlate with indices of a referring resource, use:
│     aws_cloudwatch_log_group.osquery-perf[count.index]
```

The default right now is that these will be created, but with 0 desired count, so it didn't feel too bad.

Reply:
I suppose I could move everything to a new tf file, so it's clear what's what?